### PR TITLE
Vol1 vol2 iti dsub cp iti 1315

### PR DESCRIPTION
**Pull request by Ben-Levy-62**

### CODE REVIEW

**Comment by Ben-Levy-62** [comment]
Please check Table 26.5-1 carefully.  Row heights are specified in there because of copy/paste.  Not sure they should be there.  Please also make sure I did the rowspans correctly.  Thanks!!

**Comment by JohnMoehrke** [comment]
supports #491

File: ITI/TF/Volume1/ch-26.html [hunk 1]
```diff
@@ -328,15 +328,156 @@ <h5 id="26.4.2.5.2">26.4.2.5.2 Workflow Id subscription Process Flow</h5>
       </div>
       <h2 id="26.5">26.5 DSUB Security Considerations</h2>
       <p>
-        <span>The risk analysis for this profile enumerates assets, threats, and mitigations. The risk assessment spreadsheet is stored and available from IHE at</span>
-        <a href="http://wiki.ihe.net/images/4/46/DSUB_risk_assesment.xls">http://wiki.ihe.net/images/4/46/DSUB_risk_assesment.xls</a>
-        <span>.</span>
+        <span>The risk analysis for this profile enumerates assets, threats, and mitigations.</span>
       </p>
       <p>
```
Comment (gregoriocanal) [inline ITI/TF/Volume1/ch-26.html:333]: this paragraph 333-338 should be below the table.

File: ITI/TF/Volume1/ch-26.html [hunk 2]
```diff
@@ -328,15 +328,156 @@ <h5 id="26.4.2.5.2">26.4.2.5.2 Workflow Id subscription Process Flow</h5>
...
+      </p>
+      <p>
+	The risk assessment for DSUB is presented in the table below:     
+      </p>
+      <p style="font-size:11pt">
+	      <b>Table 26.5-1: DSUB risk assessment</b>
+      </p>
+      <table cellspacing="0" cellpadding="0">
+        <thead>
+          <tr>
```
Comment (gregoriocanal) [inline ITI/TF/Volume1/ch-26.html:350]: tabel heading shouldn't be bold?

File: ITI/TF/Volume2/ITI-52.html [hunk 3]
```diff
@@ -1144,8 +1144,8 @@ <h4 id="3.52.5.3">3.52.5.3 Combining topics and filter expressions</h4>
       </table>
       <h3 id="3.52.6">3.52.6 Security Considerations</h3>
       <p>
-        <span>The risk assessment for the Document Metadata Subscribe transaction is described in the risk assessment spreadsheet for the Document Metadata Subscription Profile, which is available from IHE at</span>
-        <a href="http://wiki.ihe.net/images/4/46/DSUB_risk_assesment.xls">http://wiki.ihe.net/images/4/46/DSUB_risk_assesment.xls</a><span>. The derived mitigations are as follows:</span>
+        <span>The risk assessment for the Document Metadata Subscribe transaction is described in Table 1:26.5-1:DSUB risk assessment
+		(see Section 1:26.5 DSUB Security Considerations). The derived mitigations are as follows:</span>
```
Comment (gregoriocanal) [inline ITI/TF/Volume2/ITI-52.html:1148]: i suggest to adda an html reference to 1.26.5 section

File: ITI/TF/Volume2/ITI-53.html [hunk 4]
```diff
@@ -555,8 +555,8 @@ <h6 id="3.53.4.1.4.3">3.53.4.1.4.3 SubmissionSet Notification Example (ihe:Submi
       </p>
       <h3 id="3.53.5">3.53.5 Security Considerations</h3>
       <p>
-        <span>The risk assessment for the Document Metadata Notify transaction is described in the risk assessment spreadsheet for the Document Metadata Subscription Profile, which is available from IHE at</span>
-        <a href="http://wiki.ihe.net/images/4/46/DSUB_risk_assesment.xls">http://wiki.ihe.net/images/4/46/DSUB_risk_assesment.xls</a>
+        <span>The risk assessment for the Document Metadata Notify transaction is described in Table 1:26.5-1: DSUB risk assessment (see Section 1:26.5 DSUB Security Considerations). </span>
```
Comment (gregoriocanal) [inline ITI/TF/Volume2/ITI-53.html:558]: i suggest to add an html reference to 1.26.5 section

File: ITI/TF/Volume2/ITI-54.html [hunk 5]
```diff
@@ -267,16 +267,16 @@ <h5 id="3.54.4.1.4">3.54.4.1.4 Example</h5>
       </div>
       <h3 id="3.54.5">3.54.5 Security Considerations</h3>
       <p>
-        <span>The risk assessment for the Document Metadata Publish transaction is described in the risk assessment spreadsheet for the Document Metadata Subscription Profile, which is available from IHE at</span>
-        <a href="http://wiki.ihe.net/images/4/46/DSUB_risk_assesment.xls">http://wiki.ihe.net/images/4/46/DSUB_risk_assesment.xls</a>
+        <span>The risk assessment for the Document Metadata Publish transaction is described in Table 1:26.5-1: DSUB risk assessment (see Section 1:26.5 DSUB Security Considerations). </span>
```
Comment (gregoriocanal) [inline ITI/TF/Volume2/ITI-54.html:270]: i suggest to adda an html reference to 1.26.5 section

**Review by gregoriocanal — CHANGES_REQUESTED** [review]
see comments

**Review by gregoriocanal — APPROVED** [review]
updated as suggested

**Comment by gregoriocanal** [comment]
Reviewed and updated as needed

**Pull request merged**